test/new_relic/agent/transaction_test.rb

```diff
@@ -785,16 +785,16 @@ def test_transport_duration_returned_in_seconds_when_positive
 
     def test_transport_duration_zero_with_clock_skew
       duration = -1.0
-      parent_timestamp, txn_start = make_transport_duration_timestamps duration
+      parent_timestamp, txn_start = make_transport_duration_timestamps(duration)
       txn = in_transaction { |t| t.start_time = txn_start }
       payload = stub(timestamp: parent_timestamp)
 
       assert_equal 0, txn.calculate_transport_duration(payload)
     end
 
     def make_transport_duration_timestamps duration
-      transaction_start = Time.now()
-      parent_timestamp = ((transaction_start.to_f - duration) * 1000).round
+      transaction_start = Process.clock_gettime(Process::CLOCK_REALTIME, :millisecond)
```

Comment:
`transaction_start` is used to set `txn.start_time` in the test cases. We changed the helper to set this time in milliseconds instead of seconds as it was previously. I missed this as well, but in doing so we accidentally changed the behavior of `Transaction#calculate_transport_duration`.

Thank you @tannalynn for catching this above!